### ISSUE TITLE
场景分类baseline 训练时accuracy用的不是top3?

### ISSUE THREAD

**Issue by YanWang2014** [AIChallenger/AI_Challenger#8]
`

    train_step, cross_entropy, logits, keep_prob = network.inference(features, one_hot_labels)      

    correct_prediction = tf.equal(tf.argmax(logits, 1), tf.argmax(one_hot_labels, 1))     

    accuracy = tf.reduce_mean(tf.cast(correct_prediction, "float"))
`

**Comment by AIChallenger** [comment]
@YanWang2014 训练时只是用的top1。
